### ISSUE TITLE
Update firmware from SD card via WiFi

### ISSUE THREAD

**Issue by tonycanike** [sparkfun/SparkFun_RTK_Firmware#348]
### Subject of the issue
Can't update firmware from SD card via WiFi.

### Your workbench
Facet
F/W Oct 13, 2022 RC     AND    Oct 1, 2022 RC

Tell us how to reproduce this issue. Please post any log files from serial output that may have been generated.

### Actual behavior
RC from 10/1/2022 was running on my Facet and I wanted to update to 10/13/2022 f/w to test logging.
I put the 10/13 firmware on an SD card and put the card in my Facet.  Fired up the WiFi config and tried to update the f/w.  I clicked on the "choose file" button and it LOOKED ON MY PHONE for the f/w!   Well, that's the last place I'd put the f/w, but I get the use case.

I tried it when I had the 10/13 f/w running on the Facet and the same behavior happened.

### Expected behavior
We should be able to choose a f/w file from the SD card like we can in the serial menu.   

It's not a good use of time to have to switch between the WiFi config and the serial config to do everything we need to do

If I needed to switch f/w out in the field due to a bug, the whole serial menu thing isn't available to me, so I'd be skunked for the day.  

Thanks.

**Comment by nseidle** [comment]
tldr; You can implement the same functionality by pre-downloading any firmware bins onto your phone or computer and push them over WiFi to the device when needed. Select the firmware you'd like and update over WiFi.

The original AP config page displayed a list of available firmware files. The user could select a file and change to that firmware. This basically implemented the serial menu. However, this required an SD card be present, and a user was expected to load the SD file onto the card prior to opening the AP config page. For a general user, this was asking a lot.

We then switched to directly pushing the firmware over WiFi to the ESP32 (no SD needed). While it does require that you have the firmware on the device connected to the RTK device, it requires less hoops to jump through, and presents a far less complex AP config page (removing a list of files, radio buttons, confirmation check boxes, etc).

To prevent overwhelming the 90% of users for the 10% advanced users, I'm not going to try to blend a WiFi update and SD update together into the AP config page. You can do the same effect by loading different firmware files onto the phone or device you're connecting with.
